server/sonar-server/src/test/java/org/sonar/server/qualityprofile/RuleActivatorTest.java

```diff
@@ -983,6 +983,24 @@ public void activateOnBuiltInProfile_activate_rule_on_child_profiles() {
     assertThatRuleIsActivated(grandchildProfile, rule, changes, rule.getSeverityString(), INHERITED, emptyMap());
   }
 
+  // SONAR-10473
+  @Test
+  public void activateOnBuiltInRulesProfile_resets_severity_to_default_if_not_overridden() {
```

Comment:
A similar test could be added when the parameter default value is changed on the rule.